charts/manila-csi-plugin/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: v1.18.0
+description: Manila CSI Plugin for OpenStack
+name: manila-csi
+version: 1.2.0
```

Comment:
Not sure what version the `version` field refers to. If it's supposed to be the actual version of the plugin, it's actually `0.9.0`:

https://github.com/kubernetes/cloud-provider-openstack/blob/3909137e68519733d9832ef2dabd812550ca6884/pkg/csi/manila/driver.go#L85-L89

Reply:
Note that the `v1.18.0` specifies the CPO version, and is decoupled from the plugin's version `0.9.0`. The plugin's development is moving at a different velocity than the whole CPO umbrella repo, so that's why it's beneficial to mention both.